priv/repo/migrations/20250514114428_password_rework.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Teiserver.Repo.Migrations.PasswordRework do
+  use Ecto.Migration
+
+  def up do
+    execute("UPDATE account_users SET password = data->>'password_hash'")
```

Comment:
oh, didn't know one could do that, it's neat.
Do you think it would also be possible to drop the `password_hash` field in the json blob?
If there are too many knock on effects leave it, but I think if we can remove the `password_hash` it would reduce future confusion.

Reply:
It is possible be but originally I was hoping to leave it until it's confirmed the changes work without issues. In theory a deployment of the older version should then work without major issues. I was planning to have a DB backup ready if there are issues so maybe `password_hash` could be dropped immediately...